bridge/actions/ActionContext.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <atomic>
+namespace bridge
+{
+
+class Mixer;
+class MixerManager;
+struct StreamDescription;
+class RequestLogger;
+} // namespace bridge
+
+namespace transport
+{
+class SslDtls;
+} // namespace transport
+
+namespace bridge
+{
+struct ActionContext
+{
+    ActionContext(bridge::MixerManager& mixerManager, transport::SslDtls& sslDtls)
+        : _mixerManager(mixerManager),
+          _sslDtls(sslDtls),
+          _lastAutoRequestId(0)
+    {
+    }
+    bridge::MixerManager& _mixerManager;
+    transport::SslDtls& _sslDtls;
+    std::atomic<uint32_t> _lastAutoRequestId;
```

Comment:
Why is this part of ActionContext when it is only used by ApiRequestHandler to init RequestLogger.
The name consists of three generic words, "last", "auto", "request". Does not tell reader much about what this is. It seems to be a transaction id for logging.

Reply:
Removed from this context.